.github/dependabot.yml

```diff
@@ -1,5 +1,4 @@
 ---
-
```

Comment:
There is no need to remove this line.

Reply:
@jsf9k resolved with [0b9ee35](https://github.com/cisagov/con-pca-web/pull/362/commits/0b9ee35db46f7aad3bf26c4d4c879c897e0f2235)